### ISSUE TITLE
Import/Export settings

### ISSUE THREAD

**Issue by nascentt** [williscool/CalendarNotification#29]
Now that Im able to use your apk releases I had to uninstall qrk's APK and it reminded me of how much work it is to reconfigure the app. Setting all the preferences I use and which calendars I want to handle through the app, and it got me thinking. It'd be amazing if functionality was added to export and import the app settings to an XML or similar.
It'd also make it easier to switch from stable to debug releases easily.

**Comment by williscool** [comment]
Yeah I've run into this a couple times. If you use the debug APK then you can do it over adb.

But I have it on my roadmap to make sure that part of backup gets added.

Also, now that the binaries are properly signed, you won't have to restore all of your calendars and stuff again. 

Unless you want to switch between the debug and the release me personally, I just use the debug release all the time

**Comment by williscool** [comment]
#43 should allow for device transfer backup and hopefully the cloud ones too.

as far as export import that seems more involved because would also need storage permissions
